doc/libfbdoc/CRegex.bas

```diff
@@ -88,27 +92,26 @@ namespace fb
 	destructor CRegex _
 		( _
 		)
-		
+
 		if( ctx = NULL ) then
 			exit destructor
 		end if
 		
 		_ClearSubstrlist( ctx )
 		
-		if( ctx->vectb <> NULL ) then
-			deallocate( ctx->vectb )
-			ctx->vectb = NULL
+		if( ctx->matchdata <> NULL ) then
+			pcre2_match_data_free(ctx->matchdata)
+			ctx->matchdata = NULL
+		end if
+		if( ctx->code <> NULL ) then
+			pcre2_code_free(ctx->code)
+			ctx->code = NULL
+		end if
+		if( ctx->ccontext <> NULL ) then
+			pcre2_compile_context_create( ctx->ccontext )
```

Comment:
Shouldn't this free the compile context?
`pcre2_compile_context_free( ctx->ccontext )`